### ISSUE TITLE
Balances doesn't update

### ISSUE THREAD

**Issue by chuacw** [polkadot-cloud/polkadot-staking-dashboard#2166]
When you're on the staking page for over an era, all balances doesn't update, after you've received new rewards, after the era is over.

**Comment by An0b1s** [comment]
i can confirm that. The Unclaimed Rewards on https://staking.polkadot.cloud/#/pools don't update.

**Comment by rossbulat** [comment]
Is this referring to the payout graph? Unfortunately this API is not a subscription so it does not automatically update right now. We recently moved deployed our own graphQL API that could support subscriptions in the future, but we are still in the early days here.

**Comment by chuacw** [comment]
No, I meant the arrowed area (I blanked out the area in this screenshot)
When you're staking, you get rewards every era.
The countdown on the overview page updates, but once the countdown is reached, a reward is received, but the arrowed area doesn't update.

![2024-12-17 09_14_42-Clipboard](https://github.com/user-attachments/assets/40888f1a-f185-4607-b0c2-728ac8382679)
